cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java

```diff
@@ -100,6 +101,18 @@ public static void pushPackages(String packageName, String sourceRoot, String in
         }
 
         String orgName = manifest.getName();
+        // Validate the org-name
+        if (!validateOrg(orgName)) {
+            throw createLauncherException("invalid organization name provided " + "'" + orgName + "'." + " Only " +
```

Comment:
Can join the apostrophe to the string itself.

Reply:
Fixed with a84dc85